src/graphics/text.rs

```diff
@@ -316,24 +320,94 @@ impl Text {
     }
 
     pub fn set_text(&mut self, new_text: &str, color: Color) {
-        let atlas = self.atlas.load_cached();
-        let font_map = &atlas.font_map;
         self.batch.clear();
+        let atlas = self.atlas.load_cached();
         self.batch.set_texture(atlas.font_texture.clone());
-        let mut width: f32 = 0.;
-        for c in new_text.chars() {
+        Self::draw_word(new_text, color, &atlas.font_map, 0., 0., &mut self.batch);
+    }
+
+    fn draw_word(
```

Comment:
wouldn't "insert_word" or "add_word" be a better description of what this function performs?

Reply:
no longer relevant with latest changes